images/win/toolsets/toolset-2016.json

```diff
@@ -102,28 +102,36 @@
             "name": "azurerm",
             "versions": [
                 "2.1.0",
+                "6.13.1"
+            ],
+            "zipversions": [
                 "3.8.0",
                 "4.2.1",
                 "5.1.1",
-                "6.7.0",
-                "6.13.1"
+                "6.7.0"
             ],
             "default": "2.1.0"
         },
         {
             "name": "azure",
             "versions": [
                 "2.1.0",
+                "5.3.0"
+            ],
+            "zipversions": [
```

Comment:
nit: camelCase t/o

Reply:
camel case is not followed in this json. I have added underscore to make it consistent with other keys